packages/components/src/components/contract-card/contract-card-items/contract-card-header.jsx

```diff
@@ -50,24 +65,32 @@ const ContractCardHeader = ({
         <React.Fragment>
             <div
                 className={classNames('dc-contract-card__grid', 'dc-contract-card__grid-underlying-trade', {
-                    'dc-contract-card__grid-underlying-trade--mobile': is_mobile && !multiplier,
+                    'dc-contract-card__grid-underlying-trade--mobile': is_mobile && !multiplier && !is_turbos,
                     'dc-contract-card__grid-underlying-trade--trader': !isBot(),
+                    'dc-contract-card__grid-underlying-trade--trader--turbos': is_turbos && is_open_positions,
+                    'dc-contract-card__grid-underlying-trade--trader--turbos-sold': is_turbos && is_sold,
                 })}
             >
                 <div id='dc-contract_card_underlying_label' className='dc-contract-card__underlying-name'>
                     <Icon icon={underlying ? `IcUnderlying${underlying}` : 'IcUnknown'} width={40} size={32} />
-                    <Text size='xxs' className='dc-contract-card__symbol' weight='bold'>
-                        {display_name || contract_info.display_name}
-                    </Text>
+                    {!is_open_positions && (
+                        <Text size='xxs' className='dc-contract-card__symbol' weight='bold'>
+                            {display_name || contract_info.display_name}
+                        </Text>
+                    )}
                 </div>
-                <div id='dc-contract_card_type_label' className='dc-contract-card__type'>
+                <div id='dc-contract_card_type_label' className={'dc-contract-card__type'}>
```

Comment:
```suggestion
                <div id='dc-contract_card_type_label' className='dc-contract-card__type'>
```
@hamza-deriv just a nit-pick🙂

Reply:
Done